db/008_create_photo_uploads.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS `photo_uploads` (
+  `id` bigint unsigned NOT NULL AUTO_INCREMENT,
+  `s3_prefix` varchar(500) CHARACTER SET ascii COLLATE ascii_general_ci NOT NULL,
+  `s3_key` varchar(500) CHARACTER SET ascii COLLATE ascii_general_ci NOT NULL,
+  `user_id` bigint unsigned NOT NULL,
+  `event_id` bigint unsigned DEFAULT NULL,
+  `created_at` timestamp NOT NULL ON UPDATE CURRENT_TIMESTAMP,
+  `deleted_at` datetime DEFAULT NULL,
```

Comment:
should we add is_deleted col?

Reply:
This table is generic table for storing any photo uploads. This is required so that photos can be uploaded before the checkin is created. The mapping with checkin will be stored in a different table when the checkin is submitted.

Here, there is no need for deleted flag for now. We will keep a deleted flag with checkin.